.github/workflows/build-package-test.yml

```diff
@@ -74,7 +75,7 @@ jobs:
           POSTGRES_VERSION: ${{ matrix.POSTGRES_VERSION }}
 
       - name: Clone tools repo for test
-        run: git clone -b v0.8.22 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b ubuntu_kinetic_support --depth=1  https://github.com/citusdata/tools.git tools
```

Comment:
will be changed after merge of PR https://github.com/citusdata/tools/pull/263

Reply:
Don't forget to update this line before merge.